nifi-toolkit/nifi-toolkit-tls/src/main/java/org/apache/nifi/toolkit/tls/util/TlsHelper.java

```diff
@@ -215,4 +210,17 @@ public static JcaPKCS10CertificationRequest generateCertificationRequest(String
         JcaContentSignerBuilder jcaContentSignerBuilder = new JcaContentSignerBuilder(signingAlgorithm);
         return new JcaPKCS10CertificationRequest(jcaPKCS10CertificationRequestBuilder.build(jcaContentSignerBuilder.build(keyPair.getPrivate())));
     }
+
+    public static Extensions createDomainAlternativeNamesExtensions(String domainAlternativeNames) throws IOException {
+        List<GeneralName> namesList = new ArrayList<>();
+        for(String alternativeName : domainAlternativeNames.split(",")) {
+            namesList.add(new GeneralName(GeneralName.dNSName, alternativeName));
+        }
+
```

Comment:
Since we're on Java 8 may be we should start embracing it a bit more, at least for simple things like the above
```
List<GeneralName> namesList = Stream.of(domainAlternativeNames.split(","))
                    .map(alternativeName -> new GeneralName(GeneralName.dNSName, alternativeName))
                    .collect(Collectors.toList());
```